tsconfig.json

```diff
@@ -1,13 +1,14 @@
 {
     "compilerOptions": {
-        "target": "es6",
+        "target": "ES2015",
```

Comment:
Why do this? These are the same afaik,

Reply:
Consistency. We had a reference to es2015 and es6 in the same file, which can be somewhat confusing.

Official TS docs also says the "allowed" string is capitalized, so might as well do an easy clean-up.